opslevel/provider.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
-	"github.com/opslevel/opslevel-go/v2022"
+	opslevel "github.com/opslevel/opslevel-go/v2022"
 	"log"
```

Comment:
goland kept removing this from every file I modified for some reason until I added opslevel

Reply:
ya that means something is messed up with your environment or go.mod.

Lets make sure this is not pulled forward.  If you need help fixing your goland lets sync

Reply:
Ok, I'll take a quick look this morning to see if I can find out what's going on with my env.